SubmittingPatches.rst

```diff
@@ -294,16 +294,10 @@ A: The target branch depends on the nature of your change:
    the master branch will contain your fix as well.
 
    If you are fixing a bug (see above) *and* the bug exists in older stable
-   branches (for example, the "hammer" or "infernalis" branches), then you
-   should file a Redmine ticket describing your issue and fill out the
-   "Backport: <branchname>" form field. This will notify other developers that
-   your commit should be cherry-picked to one or more stable branches. Then,
-   target the "master" branch in your pull request.
-
-   For example, you should set "Backport: jewel, kraken" in your Redmine ticket
-   to indicate that you are fixing a bug that exists on the "jewel" and
-   "kraken" branches and that you desire that your change be cherry-picked to
-   those branches after it is merged into master.
+   branches (for example, the "hammer" or "infernalis" branches), your initial
+   PR should still target the master branch. Please see the `BACKPORTING`_
+   section, below, for information on how to ensure your bugfix gets
+   backported.
```

Comment:
@smithfarm This is the paragraph (nit) I mentioned in the chat. Dropping that colon `nature of your change:` would avoid the unnecessary quotation line.

Reply:
Done